lms/urls.py

```diff
@@ -626,6 +626,17 @@
         url(r'^certificates/html', 'certificates.views.render_html_view', name='cert_html_view'),
     )
 
+BADGE_SHARE_TRACKER_URL = url(
+    r'^certificates/badge_share_tracker/{}/(?P<network>[^/]+)/(?P<student_username>[^/]+)/$'.format(
```

Comment:
@Kelketek nit: do I understand right that this endpoint is to be used to share the badges? If it is, it might make a better URL if `badge_share_tracker` is replaced with something more intuitive (for end-user), e.g.

    r'^certificates/share_badge/{}/(?P<network>[^/]+)/(?P<student_username>[^/]+)/$'

One more thing: url formats like "{}{}/{}/{}/{}" aren't very informative. "{server}{endpont}/{course}/{user}/{badge}" are way better, IMO. So I suggest adding adding a name to format placeholder:

    r'^certificates/badge_share_tracker/{course_id}/(?P<network>[^/]+)/(?P<student_username>[^/]+)/$'

Even if it is the established way of formatting urls.